docs/index.md

```diff
@@ -93,12 +93,31 @@ Querying data:
 
 # Google.Logging.V2 (alpha)
 
-A library for working with [Stackdriver Logging](https://cloud.google.com/logging/).
+A library for working with [Google Stackdriver Logging](https://cloud.google.com/logging/).
+
+## Log4net integration - Google.Logging.Log4Net (NuGet) (alpha)
```

Comment:
Righto. I'll revise all of these bits when everything is in beta and on nuget.org anyway.